addon/styles/_frost-list-header.scss

```diff
@@ -2,11 +2,29 @@
   display: flex;
   flex-direction: row;
   align-items: center;
-  height: 45px;
 
   &.paged {
     padding-right: $frost-list-scroll-rail-gutter-width;
   }
+
+  .frost-list-header-sort-title {
+    color: $frost-color-grey-4;
+    font-size: $frost-font-s;
+    white-space: nowrap;
+  }
+
+  .frost-sort {
```

Comment:
Should this css change go into sort?

Reply:
Ya, I can put it into frost-sort

Reply:
@quincyle  Move changes into ciena-frost/ember-frost-sort#75